### PR TITLE
feat: apply changes of design to filter

### PR DESCRIPTION
**Pull request by kboutsika**
## Description

+ Filter should have 270px max-width and every text that exceeds this width should be `text-overflow: ellipsis`
+ Visited background color should be white
+ elevated mode should not have shadow when its visited

[DS-235](https://orfium.atlassian.net/browse/DS-235)

## Screenshot
Στιγμιότυπο 2021-08-04, 2 30 33 μμ
Στιγμιότυπο 2021-08-04, 2 30 49 μμ

### CODE REVIEW

**Comment by panvourtsis** [comment]
two last bullets are being fixed here https://github.com/Orfium/orfium-ictinus/pull/342

**Comment by kboutsika** [comment]
> two last bullets are being fixed here #342
> 
> ```
> Visited background color should be white
> elevated mode should not have shadow when its visited
> ```

ok! thank you! so do you want me to remove them from here?

Also do you have any idea why the tests fail?

**Comment by Christos-Zacharopoulos** [comment]
> Also do you have any idea why the tests fail?

Did you push something with `--no-verify`? 

Because some of the tests should have failed locally when you committed them. You need to update the snapshots. 

`yarn test -u`

File: src/components/Filter/Filter.tsx [hunk 1]
```diff
@@ -113,14 +114,14 @@ const Filter: React.FC<Props> = props => {
             open,
             styleType,
             hasSelectedValue,
-          })}
+          })(theme)}
```
Comment (Christos-Zacharopoulos) [inline src/components/Filter/Filter.tsx:117]: You don't have to pass theme as a prop to a style function

File: src/components/Filter/Filter.tsx [hunk 2]
```diff
@@ -113,14 +114,14 @@ const Filter: React.FC<Props> = props => {
...
           onClick={handleOpen}
           disabled={disabled}
         >
           <span css={buttonSpanStyle()}>
-            <span css={childrenWrapperStyle()}>
-              <span css={labelSpanStyle(open, hasSelectedValue)}>
-                {label}:<span>{selectedItem?.label ?? defaultValue.label}</span>
+            <span css={childrenWrapperStyle()(theme)}>
```
Comment (Christos-Zacharopoulos) [inline src/components/Filter/Filter.tsx:122]: Same here

**Comment by kboutsika** [comment]
> > Also do you have any idea why the tests fail?
> 
> Did you push something with `--no-verify`?
> 
> Because some of the tests should have failed locally when you committed them. You need to update the snapshots.
> 
> `yarn test -u`

no...and and run `yarn test -u` and test do not fail locally.

I will close this MR in an case and open another on top of this [https://github.com/Orfium/orfium-ictinus/pull/342](https://github.com/Orfium/orfium-ictinus/pull/342)

Thank you very much for your time!

**Pull request closed without merging**